website/static/js/filepage/index.js

```diff
@@ -363,6 +363,64 @@ var FileViewPage = {
         }, 1000);
 
         self.mfrIframeParent = $('#mfrIframeParent');
+        function goToRevisions(e){
+          var editable = self.editor && self.editor.selected;
+          var viewable = self.mfrIframeParent.is(':visible');
+          if (editable || viewable){
```

Comment:
I think the if statement is made to uncheck the tabs that are not being used. So if I am moving things to the revisions tab I need this if to make sure that the views and edits tab are not going to stay highlighted.